### PR TITLE
Note about issue with cascaded WFS and GeoJSON

### PR DESCRIPTION
**Pull request by JakobMiksch**
references to this issue: https://osgeo-org.atlassian.net/browse/GEOS-10021#
Adds information about an unwanted behavior of the cascaded WFS. 

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or not applicable.**

For all pull requests:

- [ ] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md) 
- [ ] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the `main` branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by Continuous Integration after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by Continuous Integration after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

### CODE REVIEW

File: doc/en/user/source/data/cascaded/wfs.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ To connect to an external WFS, it is necessary to load it as a new datastore.  T
    * - :guilabel:`Enabled`
      - Enables the store.  If disabled, no data from the external WFS will be served.
    * - :guilabel:`GET_CAPABILITIES_URL`
-     - URL to access the capabilities document of the remote WFS.
+     - URL to access the capabilities document of the remote WFS. Providing `&version=1.0.0` or `&version=1.1.0` is required for beeing able to publish GeoJSON format. See `respective Issue <https://osgeo-org.atlassian.net/browse/GEOS-10021#>`_.
```
Comment (bradh) [inline doc/en/user/source/data/cascaded/wfs.rst:32]: being is spelt incorrectly.

```suggestion
     - URL to access the capabilities document of the remote WFS. Providing `&version=1.0.0` or `&version=1.1.0` is required to able to publish GeoJSON format. See `respective Issue <https://osgeo-org.atlassian.net/browse/GEOS-10021#>`_.

```

**Review by bradh — CHANGES_REQUESTED** [review]
One typo.

I am dubious about this change at all, until its actually confirmed on JIRA.

**Review by mprins — CHANGES_REQUESTED** [review]
I don't see any value in linking to a Jira ticket that describes a problem a user has and not a solution. 
Documentation should provide a description on what should be done. From the problem description it is not even clear if the upstream WFS produces a valid capabilities document.

Also this sets up documentation to go stale.

**Comment by aaime** [comment]
Agree... Jira has 866 open tickets marked as "bug". Many are not confirmed, but still, I'm guessing a good percentage are legit, with varying degrees of severity and likeliness to stumble into them.

We cannot set a precedent to alter the documentation for every issue found, it would be un-manageable. I'd make an exception for cases where the developers have decided, for good reason, not to fix a given issue, then maybe the explanation would fit as documentation (open to debate).

This particular issue, if confirmed legit, just needs developer time/funding to be fixed. Belongs in Jira.

That said... wondering if we should link JIra from the docs sidebar, as it still is a good source of information... Something like this:
image

**Comment by ianturton** [comment]
On Fri, 16 Apr 2021 at 09:08, Andrea Aime ***@***.***> wrote:

> Agree... Jira has 866 open tickets marked as "bug". Many are not
> confirmed, but still, I'm guessing a good percentage are legit, with
> varying degrees of severity and likeliness to stumble into them.
>
> We cannot set a precedent to alter the documentation for every issue
> found, it would be un-manageable. I'd make an exception for cases where the
> developers have decided, for good reason, not to fix a given issue, then
> maybe the explanation would fit as documentation (open to debate).
>
> This particular issue, if confirmed legit, just needs developer
> time/funding to be fixed. Belongs in Jira.
>

I've already said it is likely a bug/enhancement request but it is in an
unsupported GeoTools module and I for one don't feel like fixing it in my
free time. There is no really compelling reason to set the output format to
anything other than GML anyway.

That said... wondering if we should link JIra from the docs sidebar, as it
> still is a good source of information... Something like this:
> [image: image]
> <https://user-images.githubusercontent.com/325433/114993003-a4fb8a80-9e9b-11eb-9332-1fc2cf51f65b.png>
>

That is probably a good idea.

Ian

**Comment by aaime** [comment]
Superceded by https://github.com/geoserver/geoserver/pull/4984

**Pull request closed without merging**